src/styles/index.scss

```diff
@@ -1,3 +1,55 @@
 body {
   margin: 0;
 }
+
+$days: (
+  mon: 1,
+  tue: 2,
+  wed: 3,
+  thu: 4,
+  fri: 5,
+  sat: 6,
+  sun: 7,
+);
+$month-length: 31;
+$start-day: sun;
+
+.calendar {
+  width: 707px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+  margin-top: 72px;
+  margin-inline: 149px;
+
+  @each $name, $pos in $days {
+    @if $name == $start-day {
+      &__day:first-child {
+        margin-left: calc((#{$pos} - 1) * (100px + 1px));
+      }
+    }
+  }
```

Comment:
The @each loop is correctly used to apply styles conditionally based on the start day. Ensure that the logic aligns with the intended calendar layout.